### PR TITLE
initials program and assignment1 questions added

### PR DESCRIPTION
**Pull request by ak3962**
Ashank Kumar: Assignment1 and Initials program

### CODE REVIEW

File: assignment1.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Ashank Kumar Assignment 1
+
+1. A compiler is a program that converts code written in one programming language such as C into another proggramming language, such as assembly.
+
+2a. This is a compilation error since there is a missing semicolong after 2, therefore the code wil not compile into a executable.
```
Comment (oldclesleycode) [inline assignment1.txt:5]: Right!

File: assignment1.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+2b. This is a compilation error since a variable name cannot being with a number, therefore the code wil not compile into a executable.
```
Comment (oldclesleycode) [inline assignment1.txt:7]: Perfect!

File: assignment1.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+2c. This is a runtime error since although the code will compile into an executable the computer cannot divide by zero leading to a divide bt zero error.
```
Comment (oldclesleycode) [inline assignment1.txt:9]: Right, perfect!

File: assignment1.txt [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
```
Comment (oldclesleycode) [inline assignment1.txt:10]: you forgot to comment on the difference between a run time and compile time error!

File: assignment1.txt [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+3. The method is designated as void, but a zero is still unnecessarily being returned.
```
Comment (oldclesleycode) [inline assignment1.txt:11]: this is true! but you're missing a couple of things - see if you can find them and let me know.

File: assignment1.txt [hunk 6]
```diff
@@ -0,0 +1,11 @@
+Ashank Kumar Assignment 1
+
+1. A compiler is a program that converts code written in one programming language such as C into another proggramming language, such as assembly.
```
Comment (oldclesleycode) [inline assignment1.txt:3]: sort of, more specifically, a compiler takes your code written in C (a higher level language), removes the whitespace, places this into a parse tree, optimizes it, and then takes that parse tree and translates it into assembly code.

File: initials.c [hunk 7]
```diff
@@ -0,0 +1,12 @@
+/* Ashank Kumar */
+#include <stdio.h>
```
Comment (oldclesleycode) [inline initials.c:2]: Perfect! My only suggestion is to include a small description of what this program does in your comment at the top!

File: assignment1.txt (hunk 5 above)
Comment (ak3962) [inline assignment1.txt:11]: Your using %d to print out a float when you should be using %f

**Pull request closed without merging**